src/connectors/google.sheets/functions/delete_spreadsheet_row.ts

```diff
@@ -0,0 +1,46 @@
+/** This file was autogenerated. Follow the steps in src/schema/slack/functions/README.md to rebuild **/
+import { DefineConnector } from "../../../deps.ts";
+import { Schema } from "../../../deps.ts";
+
+export default DefineConnector({
+  callback_id: "A04MG80N5CY#/functions/delete_spreadsheet_row",
+  title: "Delete from a spreadsheet",
+  description: "Delete a row in a Google Sheets spreadsheet",
+  input_parameters: {
+    properties: {
+      spreadsheet_id: {
+        type: Schema.types.string,
+        description: "Select a spreadsheet",
+        title: "Spreadsheet",
+      },
+      sheet: {
+        type: Schema.types.object,
+        description: "Select a sheet",
+        title: "Sheet",
+      },
+      column_name: {
+        type: Schema.types.string,
+        description: "Select a column",
+        title: "Choose a column to search",
+      },
+      cell_value: {
+        type: Schema.types.string,
+        description: "Add a value",
+        title: "Cell value to find",
+      },
+      google_access_token: {
+        type: Schema.slack.types.oauth2,
+        description: "Which account should we use to update the spreadsheet?",
+        title: "Google Access Token",
+      },
+    },
+    required: [
+      "spreadsheet_id",
+      "sheet",
+      "column_name",
+      "cell_value",
+      "google_access_token",
+    ],
+  },
+  output_parameters: { properties: {}, required: [] },
```

Comment:
Not sure if this field should be empty ❓

Reply:
It's fine, either should work. I believe we have tests in the deno-slack-sdk repo that allows for this to be empty.

Reply:
How does a developer know if a `delete_spreadsheet_row` step was successful — and how do they know if it was not?

Reply:
I think Slackbot will tell them that the `delete_spreadsheet_row` function returned an error.